pyaedt/filtersolutions.py

```diff
@@ -0,0 +1,60 @@
+from pyaedt.filtersolutions_core.attributes import Attributes
```

Comment:
@ramin4667 We should rename the file name to: **filter_solutions**. We are in the middle of a refactoring of some names:
https://peps.python.org/pep-0008/

The class name, for instance, is perfect: FilterSolutions

Reply:
@Samuelopez-ansys The reason to keep the filtersolutions as a single word is that FilterSolutions is the name of the application, see the (https://www.ansys.com/products/electronics/ansys-nuhertz-filtersolutions) link. To be consistent with the product name the folder and class names are saved as the application name.